firefox_ios/firefox_ios.model.lkml

```diff
@@ -47,7 +48,7 @@ explore: counters {
   }
 }
 
-explore: client_counts {
+explore: +client_counts {
```

Comment:
What does this do?

Reply:
This extends the existing explore we're importing from `looker-hub`. Before it was a new explore, not it's updating the fields of the existing explore.